pkg/operator/controller/machineconfig/controller_test.go

```diff
@@ -16,11 +16,11 @@ limitations under the License.
 
 package machineconfigcontroller
 
+/*
```

Comment:
Do we wanna remove this one?

Reply:
Since there are new changes added, thought to make use of unit test story to add new cases and existing cases also require changes, so commented those, to rework everything at once.